tests/commands/deploy.spec.ts

```diff
@@ -87,7 +87,6 @@ describe('balena deploy', function () {
 		api.expectGetRelease();
 		api.expectGetUser();
 		api.expectGetService({ serviceName: 'main' });
-		api.expectPostService409();
```

Comment:
Previous `balena-compose` own baked 'getOrCreate' was actually a `createOrGet` (see: https://github.com/balena-io-modules/balena-compose/pull/54/files#diff-f061fd1a4a8154a1ce81603ac2d2bdd64d98b27b59757f802e391bfc25581b15L102-L118) . New approach is following pinejs-client `getOrCreate` (see: https://github.com/balena-io-modules/balena-compose/pull/54/files#diff-1b4d40e2938c5e5f9c316bc5dca572406d9285437650669024f871882a053cd2R211 and https://github.com/balena-io-modules/pinejs-client-js/blob/master/src/index.ts#L1562-L1577)

Therefore, this call doesn't happen anymore and does not need to be mocked